routes/login-routes.js

```diff
@@ -0,0 +1,37 @@
+const db = require("../models");
+
+module.exports = function(app){
+
+
+    // POST new user
+    app.post("/api/users", function(req, res){
+        db.Products.create(req.body).then(function(dbUser){
+            res.json(dbUser)
+        });
+    });
+
+    // DELETE user
+    app.delete("/api/user/:id", function(req, res) {
+        db.Products.destroy({
+        where: {
+            id: req.params.id
+        }
+        }).then(function(dbUser) {
+        res.json(dbUser);
+        });
+    });
+
+    // UPDATE user info
+    app.put("/api/user", function(req, res) {
```

Comment:
Avoid passing in the whole body incase it's api changes in the future, always grab off exactly what you need.
Use es6 object initialization like I showed in the other comments.

{
  req.body.property1,
  req.body.property2,
  req.body.property3
}